docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+app:
+  extends:
+    file: docker/common.yml
+    service: app
+  links:
+    - db
+  command: rackup -s puma -p 4567 --host 0.0.0.0
+  ports:
+    - "4567:4567"
```

Comment:
Just remember to update the GitHub OAuth part in the docs ([here](https://github.com/exercism/exercism.io/blob/master/CONTRIBUTING.md)) that refers to port 4567.